cyclonedx_py/parser/environment.py

```diff
@@ -43,19 +44,26 @@
 else:
     from importlib_metadata import metadata, PackageMetadata as _MetadataReturn
 
-from cyclonedx.model import LicenseChoice
+from cyclonedx.model import License, LicenseChoice
 from cyclonedx.model.component import Component
 from cyclonedx.parser import BaseParser
 
 
+@enum.unique
+class LicenseOutputFormat(enum.Enum):
```

Comment:
:x: 
as this is a global config in the CLI, 
this class definition should be soomwhere centalized - like in `parser/__init__.py`